src/config.zig

```diff
@@ -115,7 +115,7 @@ const ConfigProcess = struct {
     client_replies_iops_read_max: usize = 1,
     client_replies_iops_write_max: usize = 2,
     tick_ms: u63 = 10,
-    rtt_ms: u64 = 300,
+    rtt_ms: u64 = 50,
```

Comment:
Is it intentional that we decrease Client request_timeout with this as well?

Reply:
This line

https://github.com/tigerbeetle/tigerbeetle/blob/8fb15c76e3fe294d5846a95405c521174620b107/src/vsr/client.zig#L152-L153

Reply:
Ah, partly. The request_timeout actually uses RTT twice.. so half of that was intentional :thinking: